core/server/master/src/main/java/alluxio/master/file/meta/Inode.java

```diff
@@ -369,6 +385,20 @@ public T setMode(short mode) {
     return getThis();
   }
 
+  /**
+   *
+   * @return the default ACL associated with this inode
+   * @throws UnsupportedOperationException if the inode is a file
+   */
+  public abstract AccessControlList getDefaultACL() throws UnsupportedOperationException;
```

Comment:
Currently, it looks like this can be `null`. This should be documented if we are going to continue to use `null` as empty.

It might be good to not use `null` for empty.